test/verify/check-metrics

```diff
@@ -760,6 +761,40 @@ class TestCurrentMetrics(MachineCase):
         b.wait(lambda: float(b.text("#load-avg .pf-l-flex div:first-child").split()[-1].rstrip(',')) > 15)
         m.execute("systemctl stop load-hog 2>/dev/null || true")  # ok to fail, as the command exits by itself
 
+        # Test podman containers
+        if m.image != "fedora-coreos":
+            container_name = "cpu-hog"
+            m.execute(f"podman run --rm -d --name {container_name} quay.io/libpod/busybox /bin/dd if=/dev/urandom of=/dev/null")
+
+            container_sha = m.execute(f"podman inspect --format '{{{{.Id}}}}' {container_name}").strip()
+            shortid = container_sha[:12]
+
+            b.wait_in_text("#current-metrics-card-cpu", f"pod {shortid}")
+            b.wait(lambda: topServiceValue(self, "Top 5 CPU services", "%", 1) > 70)
+            m.execute(f"podman stop -t 0 {container_name}")
+
+            # copy images for user podman tests; podman insists on user session
+            m.execute("""
+loginctl enable-linger $(id -u admin)
```

Comment:
You shouldn't need the linger here -- This happens during a running cockpit session for admin, i.e. admin's user bus should already be running.

Reply:
linger was dropped.